qemu/tests/cfg/qemu_img_convert_with_rate_limit.cfg

```diff
@@ -0,0 +1,39 @@
+- qemu_img_convert_with_rate_limit:
+    virt_test_type = qemu
+    type = qemu_img_convert_with_rate_limit
+    start_vm = no
+    kill_vm = yes
+    force_create_image = no
+    guest_temp_file = "/var/tmp/convert.tmp"
+    md5sum_bin = "md5sum"
+    Windows:
+        guest_temp_file = "C:\testfile"
+        x86_64:
+            sync_bin = WIN_UTILS:\Sync\sync64.exe /accepteula
+        i386, i686:
+            sync_bin = WIN_UTILS:\Sync\sync.exe /accepteula
+    convert_source = ${images}
+    variants:
+        - rate_limit:
+            required_qemu = [5.2.0-1, )
```

Comment:
move this to global ?  for this case is designed for rate limit, if version < 5.2.0-1, no need to run.

Reply:
Agree, I will update it. Many thanks.